### DISCUSSION TITLE
Vertical Screen Support

### DISCUSSION THREAD

**Discussion by Aru1199** [GitSquared/edex-ui#1166] [Ideas]
I only did a quick search in the discussions here on Github, so I'm sorry if this has been mentioned before, but I'd just like to mention that it would be cool if edex-ui supported vertically oriented monitors. At the moment, everything just overlaps with each other when launching it in a vertical screen. Thanks!

**Comment by ghost** [comment]
If you know some CSS, you can use the `injectCSS` key to reorganize panels.  See, e.g. the `cyborg-focus` theme.
